### PR TITLE
add required state parameter to acme_certificate route53 example

### PR DESCRIPTION
**Pull request by endorama**
##### SUMMARY

route53 module state parameter is required (https://docs.ansible.com/ansible/latest/modules/route53_module.html#parameters), so the example is not working out of the box in ansible.

Required state parameters has been introduced in commit 698fa37a4496b44b32c2b851f0cf83bd664e8eda (https://github.com/ansible/ansible/commit/698fa37a4496b44b32c2b851f0cf83bd664e8eda)

Thank you!

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

- `route53`

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/modules/crypto/acme/acme_certificate.py [hunk 1]
```diff
@@ -244,6 +245,7 @@
 #     record: "{{ item.key }}"
 #     type: TXT
 #     ttl: 60
+#     state: present
 #     # Note: item.value is a list of TXT entries, and route53
 #     # requires every entry to be enclosed in quotes
 #     value: "{{ item.value | map('regex_replace', '^(.*)$', '\'\\1\'' ) | list }}"
```
Comment (felixfontein) [inline lib/ansible/modules/crypto/acme/acme_certificate.py]:
```suggestion
#     value: "{{ item.value | map('regex_replace', '^(.*)$', '\"\\1\"' ) | list }}"
```
(I don't know why github thinks the original line has a leading space...)

File: lib/ansible/modules/crypto/acme/acme_certificate.py [hunk 2]
```diff
@@ -233,6 +233,7 @@
 #     record: "{{ sample_com_challenge.challenge_data['sample.com']['dns-01'].record }}"
 #     type: TXT
 #     ttl: 60
+#     state: present
 #     # Note: route53 requires TXT entries to be enclosed in quotes
 #     value: "{{ sample_com_challenge.challenge_data['sample.com']['dns-01'].resource_value }}"
```
Comment (felixfontein) [inline lib/ansible/modules/crypto/acme/acme_certificate.py]: I think this should be:
```suggestion
#     value: "{{ sample_com_challenge.challenge_data['sample.com']['dns-01'].resource_value | regex_replace('^(.*)$', '\"\\1\"') }}"
```
(I don't know why github thinks the original line has a leading space...)

**Review by felixfontein — APPROVED** [review]
Thanks for spotting and fixing this!

I added two suggestions of how #49032 can also be fixed by this; could you add this to your PR?

**Comment by felixfontein** [comment]
(Anyway, the `state` parameter of `route53` is older than this example, so it has always been wrong... I adjusted it from a [working example](https://github.com/felixfontein/acme-certificate/blob/master/tasks/dns-route53-create.yml) I have, and I guess I screwed the "adjustment" up... :) )

**Review by felixfontein — APPROVED** [review]
Thanks a lot for fixing this!

LGTM

**Comment by endorama** [comment]
@felixfontein thank you!

**Review by felixfontein — APPROVED** [review]
shipit

**Comment by resmo** [comment]
shipit

**Review by resmo — APPROVED** [review]

**Comment by felixfontein** [comment]
@endorama thanks again for the PR!
@resmo thanks for reviewing and merging!

**Pull request merged**